src/termination.ml

```diff
@@ -0,0 +1,378 @@
+(** Temination checking.
+    This module allows the translation of a signature in the TPDB format used
+    in the termination competition, see Termination Problem Database:
+    http://cl2-informatik.uibk.ac.at/mercurial.cgi/TPDB
```

Comment:
It would be better to give a link to the XTC format itself: http://cl2-informatik.uibk.ac.at/mercurial.cgi/TPDB/raw-file/tip/xml/xtc.xsd